apps/web/src/api/v2/builds/update.ts

```diff
@@ -128,6 +128,11 @@ const handleUpdateParallel = async ({
 
   if (complete) {
     await buildJob.push(build.id);
+    const account = await Project.relatedQuery<Account>("account")
```

Comment:
You are in a `transaction` and not passing `trx`. It should break tests.

Reply:
Actually this line is out of transaction so I don't need trx. But do you think it should be included in a transaction? By the way, do you think there is a better place to call stripe update ?

Reply:
Yeah I think the build process before marking the build "complete" is a better place. What do you think?